cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperatorTest.java

```diff
@@ -246,6 +255,55 @@ private void krNewToProposalReady(VertxTestContext context, int pendingCalls, Cr
                 }));
     }
 
+    private void krNotReadyToProposalReadyOnSpecChange(VertxTestContext context, int pendingCalls, CruiseControlEndpoints endpoint, KafkaRebalance kr) throws IOException, URISyntaxException {
```

Comment:
Why is this not inlined int he test? It seems to be used only by one test anyway. So no code is shared here.